File: ms-factura-flask/factura.py
import json

class Factura():
    def __init__(self, id, date, value, client_id, client_name):
        self.id = id
        self.date = date
        self.value = value
        self.client_id = client_id
        self.client_name = client_name
    
    def toJSON(self):
        return json.dumps(self, default=lambda o: o.__dict__, 
            sort_keys=True, indent=4)

File: camel-route/src/main/java/ec/insuasti/ups/homeworks/camelroute/Factura.java
package ec.insuasti.ups.homeworks.camelroute;

import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@ToString
@NoArgsConstructor
public class Factura {
    private int id;
    private String date;
    private float value;
    private String client_id;
    private String client_name; 


}
